creator-node/default-config.json

```diff
@@ -41,5 +41,6 @@
   "dataProviderUrl": "http://docker.for.mac.localhost:8545",
   "dataNetworkId": "",
   "userMetadataNodeUrl": "http://docker.for.mac.localhost:4000",
-  "rehydrateMaxConcurrency": 10
+  "rehydrateMaxConcurrency": 10,
+  "creatorNodeEndpoint": ""
```

Comment:
update all warnings etc and behavior to consider this the same as 'Null' - an empty string should indicate no config